### PR TITLE
Client-side code for image deletion

### LINKED ISSUE #1770: Allow deleting images via cli
Create a command `neuro image rm URI` for deleting images from registry.

### PR DESCRIPTION
**Pull request by anayden**
Closes #1770 

To be accompanied with AWS ECR support on the `platform-registry-api` side and e2e tests.

### CODE REVIEW

**Review by asvetlov — COMMENTED** [review]
please don't forget the API doc updating

File: neuromation/api/images.py [hunk 1]
```diff
@@ -103,6 +103,30 @@ async def push(
...
+            return resp.headers["Docker-Content-Digest"]
+
+    async def rm(self, remote: RemoteImage, digest: str) -> None:
+        try:
+            name = f"{remote.owner}/{remote.name}"
+            auth = await self._config._registry_auth()
+            url = self._registry_url / name / "manifests" / digest
+            async with self._core.request("DELETE", url, auth=auth) as resp:
+                assert resp
+        except DockerError as error:
```
Comment (asvetlov) [inline neuromation/api/images.py:126]: How is it possible to get DockerError from HTTP (aiohttp precisely) call?

**Review by asvetlov — CHANGES_REQUESTED** [review]
Looks good except for one place

File: neuromation/api/images.py (hunk 1 above)
Comment (anayden) [reply neuromation/api/images.py]: Good catch! That's a leftover from the very first draft of the code. Removed.

File: neuromation/api/images.py [hunk 2]
```diff
@@ -117,15 +117,12 @@ async def digest(self, remote: RemoteImage) -> str:
...
-            url = self._registry_url / name / "manifests" / digest
-            async with self._core.request("DELETE", url, auth=auth) as resp:
-                assert resp
-        except DockerError as error:
-            if error.status == 404:
-                raise ValueError(f"Image {remote} was not found") from error
+        name = f"{remote.owner}/{remote.name}"
+        auth = await self._config._registry_auth()
+        url = self._registry_url / name / "manifests" / digest
+        print(url)
```
Comment (asvetlov) [inline neuromation/api/images.py:123]:
```suggestion
```

**Review by asvetlov — APPROVED** [review]

**Pull request merged**